packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -171,7 +171,7 @@ class RunPanel extends React.PureComponent {
                     <Drawer
                         className={!is_mobile ? 'run-panel__container' : undefined}
                         contentClassName='run-panel__content'
-                        clear_stat_button_text={localize('Clear stat')}
+                        clear_stat_button_text={localize('Reset')}
                         footer={!is_mobile && footer}
```

Comment:
[Dervin asked to update the Button text to Reset](https://deriv-group.slack.com/archives/CKSRAP065/p1596431978429500?thread_ts=1596430829.427800&cid=CKSRAP065)